.github/workflows/ci.yml

```diff
@@ -5,43 +5,48 @@ on:
   pull_request:
```

Comment:
Will take a look at this, also probably need to cleanup the UI a bit for multifile verification. I'll use this and the above perpetual example as test cases

Reply:
You might know this, since you haven't commented on this thread yet, but: I just tested this with the latest version and it looks the same as yesterday.

Reply:
Yea sorry this is the one I was talking about in element yesterday

> interesting that this didn't match deployed code 
> 
![image](https://github.com/ScopeLift/cove-frontend/assets/17163988/3080d68d-7ea6-47b3-9daa-66550d672342)
> all comments should be resolvable except for [this one](https://github.com/ScopeLift/cove-frontend/pull/7#discussion_r1265836165) if you wanted to check them out (and I resolved the nits). Spent some time debugging but haven't gotten to the bottom of it, and it might be related to why deployed code wasn't verified in the above screenshot. So I'll get back to that likely tomorrow